assets/intercom.js

```diff
@@ -0,0 +1,7 @@
+//Set your APP_ID
+var APP_ID = '{{ .Site.Params.IntercomAppId }}'
```

Comment:
Dont you need to add this prop to the `config.tml` ?

Reply:
Hey @gonpombo8 I added the prop here: https://github.com/decentraland/documentation/pull/82/files